### PR TITLE
Ft docs to wiki

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Changes the docs links from docs to wiki

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- Do a static code analysis to ensure that the (README) changes do not affect existing functionality, and that they still communicate the same message as before.

#### Related PRs
- https://github.com/airqo-platform/AirQo-api/pull/1642

### CODE REVIEW

File: calibrate/.env.sample [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PORT=5000
 NODE_PATH=src/
 REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
-REACT_APP_DOCS_BASE_URL=https://staging-docs.airqo.net/#/
+REACT_APP_DOCS_BASE_URL=https://wiki.airqo.net/#/
```
Comment (Baalmart) [inline calibrate/.env.sample:4]: thanks @123MwanjeMike , do we also have `staging-wiki`?
If yes, shouldn't the `BASE_URL` be an ENV?

File: calibrate/.env.sample [hunk 2]
```diff
@@ -1,6 +1,6 @@
 PORT=5000
 NODE_PATH=src/
-REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
-REACT_APP_DOCS_BASE_URL=https://staging-docs.airqo.net/#/
+REACT_APP_BASE_URL=http://platform.airqo.net/api/v1/
+REACT_APP_DOCS_BASE_URL=https://wiki.airqo.net/#/
 REACT_APP_BASE_CALIBRATE_URL=
-REACT_APP_AUTHORIZATION_TOKEN=ey123abc
\ No newline at end of file
+REACT_APP_AUTHORIZATION_TOKEN=ey123abc
```
Comment (Baalmart) [inline calibrate/.env.sample:6]: @123MwanjeMike ....isn't this a security concern?

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
thanks @123MwanjeMike

**Pull request merged**